documentation/internal/Actions-on-Applications.md

```diff
@@ -46,7 +46,8 @@ The options are per section of the application, but will show on the application
 
 ### Application list view
 
-On server side we use smart view to calculate the count of each of the following (used to define available actions on the front-end) based on status of `review_assignment` and `review` (associated to self for Reviewer, and associated to any review for Assigners).
+On server side we use smart view to calculate the count of each of the following (used to define available actions on the front-end) based on status of `review_assignment` and `review`.
+The GraphQL query to application list view will fetch ALL `review_assignment` related to the current application `serial`, for all reviewer `user` of any `level` and `stage`.
```

Comment:
The GraphQL query to application list view -- might be good to include the name of the query (in brackets) here.